projects/storefrontapp/src/environments/features/b2b/b2b.feature.ts

```diff
@@ -0,0 +1,57 @@
+import { translationChunksConfig, translations } from '@spartacus/assets';
```

Comment:
It's duplicate of `projects/storefrontapp/src/environments/b2b/b2b.feature.ts`
Probably product of merging branches.
Can we remove it please?